### DISCUSSION TITLE
plotting (relative area vs. Grain area) for different samples on a one graph!

### DISCUSSION THREAD

**Discussion by nargesmbg** [mtex-toolbox/mtex#1847] [Ask Anything]
hello everyone :), 

I'd like to plot (relative area vs. Grain area) for my 5 samples on one graph, ideally showed by different colors and data label like: color blue = sample 1 

I appreciate your help!
Best 
Narges

**Comment by kilir** [comment]
Hi,

I'd say this is somethign you'll find in the Matlab help e.g. 

https://www.mathworks.com/help/matlab/ref/scatter.html
https://www.mathworks.com/help/matlab/ref/legend.html

Cheers,
Rüdiger
